coalib/results/Result.py

```diff
@@ -93,6 +115,15 @@ def __init__(self,
         self.id = uuid.uuid4().int
         self.aspect = aspect
 
+    @property
+    def message(self):
+        self._message = self.message_base.format(**self.message_arguments)
```

Comment:
if you do the .format on every get (and not in the setter) there's no need at all for storing it in `self._message` as it's not used elsewhere